python/ray/rllib/common.py

```diff
@@ -1,15 +1,44 @@
 from collections import namedtuple
 from datetime import datetime
-import json
 import logging
 import os
+import simplejson
```

Comment:
adding it to setup.py, but unfortunately we need it for NaNs, see below; it would be good to standardize around one JSON implementation though, I agree.